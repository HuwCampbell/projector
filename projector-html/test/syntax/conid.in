case foo of Bar_B_Q -> Baz