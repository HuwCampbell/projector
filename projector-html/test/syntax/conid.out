Template
  ()
  Nothing
  (TECase
     ()
     (TEVar () TId { unTId = "foo" })
     (TAlt
        ()
        (TPCon () TConstructor { unTConstructor = "Bar_B_Q" } [])
        (TEVar () TId { unTId = "Baz" }) :|
        []))